pkg/api/validation/validation.go

```diff
@@ -1784,7 +1901,7 @@ func ValidateResourceRequirements(requirements *api.ResourceRequirements, fldPat
 				limitValue = quantity.MilliValue()
 			}
 			if limitValue < requestValue {
-				allErrs = append(allErrs, field.Invalid(fldPath, quantity.String(), "limit cannot be smaller than request"))
+				allErrs = append(allErrs, field.Invalid(fldPath, quantity.String(), "must be greater than or equal to `request`"))
```

Comment:
how about just removing the backticks.  It's clear to me.

On Tue, Dec 15, 2015 at 11:20 PM, Chao Xu <notifications@github.com> wrote:

> In pkg/api/validation/validation.go
> <https://github.com/kubernetes/kubernetes/pull/18694#discussion_r47743430>
> :
>
> > @@ -1784,7 +1901,7 @@ func ValidateResourceRequirements(requirements *api.ResourceRequirements, fldPat
> >  				limitValue = quantity.MilliValue()
> >  			}
> >  			if limitValue < requestValue {
> > -				allErrs = append(allErrs, field.Invalid(fldPath, quantity.String(), "limit cannot be smaller than request"))
> > +				allErrs = append(allErrs, field.Invalid(fldPath, quantity.String(), "must be greater than or equal to `request`"))
>
> A nit. There is no field named request. so how about "must be greater
> than or equal to the counterpart in `requests`"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/18694/files#r47743430>.
>